web/components/feed/feed-items.tsx

```diff
@@ -137,7 +145,15 @@ export function FeedComment(props: {
     bought = bet.amount >= 0 ? 'bought' : 'sold'
     money = formatMoney(Math.abs(bet.amount))
   }
-  const { text, userUsername, userName, userAvatarUrl, createdTime } = comment
+  const { text, userUsername, userName, userAvatarUrl, createdTime, userId } =
+    comment
+
+  const userBets = useUserContractBets(userId, contract.id)
+  const { yesFloorShares, noFloorShares } = useSaveShares(
```

Comment:
I think it works, but each usage is overwriting the same spot in local storage. And that spot in local storage is read before the bets load, e.g. to show your shares in the bet panel.

So comments from other users are saving their yes and no shares so that when you load the page it will show their position in your bets panel (until the actual bets load).

The impact on a user is minimal (flash of incorrect shares, is probably more visible in production than in localhost). But   saving incorrect data to localstorage doesn't seem like something we should deliberately do unless it's an emergency.